benchmarking/daint_single_node/stdout_to_json.py

```diff
@@ -13,12 +13,12 @@
 # timers reported in stdout of FV3GFS, multiple entries are accumulated
 TIMER_MAPPING = {
   "total": ["Total runtime"],
-  "init": ["1-Initialization", "2-Main-loop-1st-trip"],
-  "main_loop": ["3-Main-loop"],
-  "fv_dynamics": ["3.1.1-fv_dynamics"],
-  "dyn_core": ["3.1.1.1-dyn_core"],
-  "tracer_adv": ["3.1.1.2-Tracer-advection"],
-  "remapping": ["3.1.1.3-Remapping"]
+  "initialization": ["1-Initialization", "2-Main-loop-1st-trip"],
+  "mainloop": ["3-Main-loop"],
```

Comment:
Should we switch these to MainLoop and Initialization while we're at it?

Reply:
right now the three large timer are all small (initialization, mainloop and total). It felt deliberate?